### ISSUE TITLE
Add Open in Side View in Bookmark Properties

### ISSUE THREAD

**Issue by NicholasJohn16** [mozilla/side-view#383]
Up until recently, Firefox had the option to open a bookmark in a sidebar, but this was recently removed (https://bugzilla.mozilla.org/show_bug.cgi?id=1452645). With Side View seemingly taking that features place, could the option be added to bookmarks so they could be opened in Side View with a single click? It was extremely handy before to be able to set a bookmark in the toolbar to just open in the sidebar and I frequently forget to right click and choose Open in Side View now.

**Comment by TwisterMc** [comment]
I agree. Please add this feature! It'd make the extension much handier.
